### ISSUE TITLE
Label Inputs

### ISSUE THREAD

**Issue by jpdevries** [iancover/QuizApp#1]
Should be able to click the label of a radio to check that radio. Labeling inputs is also for screen reader support and semantics.
https://dequeuniversity.com/rules/axe/2.1/label?application=axeChrome
